pkg/sbom/sbom.go

```diff
@@ -47,15 +65,17 @@ func SBOMWorkflow(
 		sbomDocs = append(sbomDocs, newData(depGraph, service.MimeTypeCycloneDXJSON, sbomBytes))
 	}
 
+	logger.Printf("Successfully generated SBOM documents: count %d\n", len(sbomDocs))
+
 	return sbomDocs, nil
 }
 
 func Init(e workflow.Engine) error {
 	flagset := pflag.NewFlagSet("snyk-cli-extension-sbom", pflag.ExitOnError)
-	// TODO: add proper user documentation.
-	flagset.String("file", "", "usage docs for file option")
-	// TODO: add proper user documentation.
-	flagset.StringP("format", "f", "text", "usage docs for format option")
+
+	flagset.Bool(flagExperimental, false, "Explicitly enable sbom command.")
```

Comment:
Is the purpose of this just to highlight that the functionality is experimental?

Reply:
this comes out of a conversation with gareth. context: https://snyk.slack.com/archives/C04DMQLF0UD/p1670404911293099?thread_ts=1670403890.327339&cid=C04DMQLF0UD

we've basically decided to roll out the feature as ungated, but undocumented, and have this flag as the final frontier, per gareth’s suggestion.